perun-core/src/main/java/cz/metacentrum/perun/core/impl/GroupsManagerImpl.java

```diff
@@ -122,9 +123,10 @@ public Group createGroup(PerunSession sess, Vo vo, Group group) throws GroupExis
 			}
 		}
 
-		// Check the group name, it can contain only a-Z0-9_- and space
-		if (!group.getShortName().matches("^[- a-zA-Z.0-9_]+$")) {
-			throw new InternalErrorException(new IllegalArgumentException("Wrong group name, group name can contain only a-Z0-9.-_: and space characters. " + group));
+		try {
+			Utils.validateGroupName(group.getShortName());
```

Comment:
This check is a safe-guard repetition of check from entry layer. Maybe we should add the same check for ``updateGroup()``.

Reply:
I have added the check also to the update group method.